lib/terminal.robot

```diff
@@ -63,7 +63,11 @@ Read From Terminal Until
     ...    connection to the DUT (Telnet or SSH).
     [Arguments]    ${expected}
     IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
-        ${output}=    Telnet.Read Until    ${expected}
+        TRY
+            ${output}=    Telnet.Read Until    ${expected}
+        EXCEPT    EOFError: telnet connection closed
+            ${output}=    FAIL    Verify and close your connections before starting tests
```

Comment:
The issue says about 3 possible ways to unblock yourself:

- telnet (or minicom on RTE) connection might be opened - either by you or someone else
- verify and close your connections before starting tests
- stop minicom processes on RTE, restart ser2net service (systemctl restart ser2net)

You have included only one of them

Reply:
oh, i thought i get to choose only one, well this is the improved version with all 3: https://github.com/Dasharo/open-source-firmware-validation/pull/382/commits/d3d58eea96a0e73c96577762859c28435e3f8a41